doc/content/components/identity-server.md

```diff
@@ -4,8 +4,80 @@ description: ""
+
+### Users
+
+The first entity registered in the Identity Server is usually the admin user. Users can register by providing an email address and choosing a user ID and password. This information can later be used to login.
+
+The user ID is the unique identifier of a user. User IDs are in the same namespace as organization IDs. This means that it is not possible to create a user with the same ID as an existing organization. For security reasons it is also not possible to re-use the ID of a deleted user or organization.
+
+Users can be "admin", which gives them elevated privileges. The normal registration process does not allow users to register as admin, which is why the [Getting Started guide]({{< ref "/guides/getting-started" >}}) creates the admin user with a different command.
+
+Users can be in one of multiple states: requested, approved, rejected, suspended, etc. The state of a user determines if the user is able to perform actions, and which actions. Normally, users are in the "approved" state. If The Things Stack is configured to require admin approval for new users, users are initially in the "requested" state, and an admin user can update them to the "approved" or "rejected" state. Users can also be suspended by admins if they misbehave.
+
+### Gateways
+
+Gateways can be registered by choosing an ID and optionally registering the EUI of the gateway. After registration, an API key can be created; the gateway can use its ID (or EUI) together with that API key to authenticate with the The Things Stack.
+
+For correct operation of the gateway, it is important to set the frequency plan ID and set whether the gateway needs to be compliant with a duty cycle.
+
+If the gateway is capable of communicating with a [Gateway Configuration Server]({{< relref "gateway-configuration-server.md" >}}), the Gateway Server address, and firmware update settings can be set in the gateway registry.
+
+The gateway registry also stores information about the antenna(s) of the gateway, such as location and gain.
+
+For more details, see the the [gateway API reference]({{< ref "/reference/api/gateway.md" >}}).
+
+### Applications
+
+Applications are used to organize registrations and traffic of multiple end devices in once place. An application typically corresponds to a collection of end devices that are in the same deployment, or of the same type.
+
+For more details, see the the [application API reference]({{< ref "/reference/api/application.md" >}}).
+
+### End Devices
+
+The end device registry in the Identity Server stores only metadata of end devices, allowing clients such as the Console and CLI to list end devices in an application. It typically stores metadata about the brand, model, hardware, firmware and location of the end device. It also stores addresses of the Network Server, Application Server and Join Server, so that clients such as the Console and CLI know where to fetch other properties of the end device.
```

Comment:
`It also stores addresses of the Network Server, Application Server and Join Server, so that clients such as the Console and CLI know where to fetch other properties of the end device.`
Nothing wrong here, however, this is not how the console fetches ns/js/as related fields for devices. The console uses addresses from the stack configuration. Should this be addressed @kschiffer ?

Reply:
Indeed, the console will not attempt to fetch from arbitrary URIs based on the `*_address` properties in the entity, which is as far as I understand also not the desired behavior.
Is there any misunderstanding here @htdvisser ?

Reply:
Ok, I reworded this so that it doesn't imply anything about fetching, just knowing where the data is stored.